test/org/zalando/stups/pierone/v2_test.clj

```diff
@@ -14,15 +15,35 @@
 
 (deftest v2-test
   (let [system (u/setup)
+        manifest-file (slurp "test/org/zalando/stups/pierone/manifest.json")
```

Comment:
Can we put the initialization of these variables in `test_utils` so that the test is readable more easily?